.mvn/maven.config

```diff
@@ -1,4 +1,4 @@
--Drevision=1.5.0-SNAPSHOT
+-Drevision=1.7.0-SNAPSHOT
```

Comment:
Why do we already need 1.7?

Reply:
should be 1.6 as we are preparing Beta 1.6 and 1.6 dev is not closed

Reply:
Done 😄